test/jdk/jdk/incubator/vector/Double128VectorTests.java

```diff
@@ -1078,6 +1111,10 @@ static long bits(double e) {
                 return fill(s * BUFFER_REPS,
                             i -> (((double)(i + 1) == 0) ? 1 : (double)(i + 1)));
             }),
+            withToString("double[i / 10.0 + 0.1]", (int s) -> {
+                return fill(s * BUFFER_REPS,
+                            i -> (double)(i / (double) 10.0 + 0.1));
+            }),
```

Comment:
Did this generate rounding issues for addition?
Another option would be random values that make sure to fill the whole mantissa with random information.
Of course the tricky part is to keep them within reasonable bounds so that on multiplication they do not degenerate to zero or infinity.
Also: it would be nice to have some cases with extreme values (infinity, NaN, etc).

Reply:
Yes, this generator generates a rounding issue for addition:
```
test FloatMaxVectorTests.ADDReduceFloatMaxVectorTests(float[i / 10.0 + 0.1]): failure
java.lang.AssertionError: at index #16 expected [39.2] but found [39.199997]
```

Reply:
I updated this generator to `0.01 + (i / (i + 1))`, plus a variant that replaces some of the elements of this sequence with values from the `cornerCaseValues` generator.
This should address all of your concerns. Almost all values in this sequence are very close to 1, so they can be added and multiplied without overflow, up to about 2000 elements for `float`s. The mantissas have bits all over the place. The exponents are very limited, but a wider range of exponents is exercised by the other tests.